### DISCUSSION TITLE
Managed Window Overflow the boundaries of tiling BSP

### DISCUSSION THREAD

**Discussion by troydm** [asmvik/yabai#755] [Ideas]
There are some Applications such as "Disk Utility" that have some fixed minimum window size meaning they can't be resized to smaller than their fixed size specified (meaning they can't be freely resized), when these windows are tiled they often overflow the boundaries of tiling mode meaning the whole tiling gets messed up, I definitely can add rules for every single application that have these kind of windows, but it would be nice to have a feature that when enabled checks if window is not within specified tiling boundaries when it is managed and resized and is overflowing it's tilling boundaries, make that window float automatically

**Comment by dominiklohmann** [comment]
I have migrated this to discussions.
